.circleci/config.yml

```diff
@@ -338,12 +261,24 @@ jobs:
 
             # setup .conbench file
             touch .conbench
-            echo "url: https://velox-conbench.voltrondata.run/" >> .conbench
+            echo "url: $CONBENCH_URL" >> .conbench
             echo "email: $CONBENCH_EMAIL" >> .conbench
             echo "password: $CONBENCH_PASSWORD" >> .conbench
             echo "host_name: CircleCI-runner" >> .conbench
 
-            conbench cpp-micro --run-name="commit: $(git rev-parse HEAD)" --run-reason="commit"
+            # figure out run name/reason
```

Comment:
We might want to add a bit more details here about why we treat `main` differently

Reply:
good idea!